java/openmldb-jdbc/src/test/java/com/_4paradigm/openmldb/jdbc/SQLRouterSmokeTest.java

```diff
@@ -560,4 +569,19 @@ public void testInsertPreparedStateBatch() {
             Assert.fail();
         }
     }
+
+    @Test
+    public void testDDLParseMethods() throws SQLException {
```

Comment:
Add test for DDLParse Fail. 
`genOutputSchema ` returns null or empty `List<Column>`?

Reply:
done